sdk/storage/generate.ps1

```diff
@@ -2,19 +2,19 @@ pushd $PSScriptRoot/Azure.Storage.Common/swagger/Generator/
 npm install
 npm install -g autorest@beta
 
-cd $PSScriptRoot/Azure.Storage.Blobs/swagger/
-autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
+#cd $PSScriptRoot/Azure.Storage.Blobs/swagger/
```

Comment:
local change.

Reply:
Yep, I should have these issues in all 4 PRs.